packages/synchro-charts/src/components/sc-dial/utils/type.ts

```diff
@@ -21,4 +21,10 @@ export type DialErrorMessages = {
 
 export type DialMessages = {
   error: DialErrorMessages;
+  loading: string;
+};
+
+export type AngleDefault = {
```

Comment:
should document whether these are in radians or degrees

Reply:
Done. Add a note.